app/models/estimated_population.rb

```diff
@@ -0,0 +1,47 @@
+class EstimatedPopulation < ApplicationRecord
+  belongs_to :region
+
+  validates :population, numericality: true, presence: true
+  validates :diagnosis, presence: true
+
+  enum diagnosis: {HTN: "HTN", DM: "DM"}
+
+  validate :can_only_be_set_for_district_or_state
+  after_commit :update_state_population
+
+  def can_only_be_set_for_district_or_state
+    unless region.district_region? || region.state_region?
+      errors.add(:region, "can only set population for a district or a state")
+    end
+  end
+
+  def is_population_available_for_all_districts
```

Comment:
Is this used anywhere?

Reply:
@rsanheim this isn't used anywhere yet, but will use for the following PR when we show the population data in the Report "Details" pages